README.md

```diff
@@ -27,6 +27,7 @@ This library allows us to:
 - merge multiple transformation matrices in a single matrix that is the **composition of multiple matrices**
 - work with strings in both directions: **parse**, **render**
 - **apply a transformation matrix to point(s)**
+- **decompose a matrix into translation, rotation and scaling components, with flip decomposition support**
```

Comment:
Can you add this line to README.template.hbs? Otherwise this row will be lost during the next build, since the README file is autogenerated.

Reply:
Sure x2